beacon-chain/p2p/info.go

```diff
@@ -0,0 +1,65 @@
+package p2p
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/sirupsen/logrus"
+)
+
+// InfoHandler is a handler to serve /p2p page in metrics.
+func (s *Service) InfoHandler(w http.ResponseWriter, _ *http.Request) {
+	buf := new(bytes.Buffer)
+	if _, err := fmt.Fprintf(buf, `bootnode=%s
+self=%s
```

Comment:
is it just my browser? or this looks messed up

Reply:
Yeah it looks off

Reply:
using ` formats it exactly how you have it. If i add tabs/spacing here, then it would be tabbed/spaced in the rendering.

Reply:
Ah k, sounds good in that case